types/tar-stream/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "@definitelytyped/dtslint/dt.json" }
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "await-promise": false
+    }
```

Comment:
Fix any errors after removing this rule. If you need to disable the rule, disable it for that line so we can evaluate the exception

```suggestion
```

Reply:
Changed in 524a61af67212726fbc37121efe177d2dcd883d6. But do you know why `tslint` is ignoring the `tsconfig.json` settings? I explicitly set `ES2020` as target and lib and also included `ESNext.AsyncIterable` and `tslint` is still complaining.